Havit.Blazor.Components.Web.Bootstrap/Chips/HxChipList.razor

```diff
@@ -9,11 +9,20 @@
                 @if (chipItem.Removable)
                 {
                     <span role="button" class="hx-chip-remove-btn"
-                        @onclick="() => HandleRemoveClick(chipItem)" @onclick:stopPropagation>
+                          @onclick="() => HandleRemoveClick(chipItem)" @onclick:stopPropagation>
                         <HxIcon Icon="BootstrapIcon.X" />
                     </span>
                 }
             </HxBadge>
         }
+        @if (Reset == true)
+        {
+            @ResetButtonTemplate
+            <span role="button" class="hx-chip-reset-btn"
+                @onclick="() => HandleResetClick()" @onclick:stopPropagation>
+                @ResetButtonText
+                <HxIcon Icon="BootstrapIcon.X" />
+            </span>
+        }
```

Comment:
How do I use `ResetButtonTemplate`?
I probably do not need to have ability to trigger the `OnResetClick` from the template (the handler can be used directly), but I will probably want to get rid of the original `span.hx-chip-reset-btn`.

+ the class should be `.hx-chip-list-reset-btn` (missing the `-list-` part)